controllers/error-handling-controllers.js

```diff
@@ -1,4 +1,22 @@
+exports.handle400Errors = (err, req, res, next) => {
```

Comment:
It may make more sense to split the error handlers by type, eg custom and psql thrown. This will make your logic tidier later on when handling errors from further endpoints.